cylc/flow/scheduler_cli.py

```diff
@@ -393,7 +393,7 @@ def _distribute(host, workflow_id_raw, workflow_id):
         cmd.append("--host=localhost")
 
         # Re-invoke the command
-        _remote_cylc_cmd(cmd, host=host)
+        cylc_server_cmd(cmd, host=host)
```

Comment:
Behaviour change: This now uses the localhost platform config.

It now also logs the command to debug level.